.eslintignore

```diff
@@ -0,0 +1,7 @@
+node_modules/
+build/
+dist/
+
+assets/
+public/
+/**/*.d.ts
```

Comment:
GitHub needs empty line at the end of each file you want to add

Reply:
Added empty line for each file (also changed settings for my IDE)